cegui/src/text/LegacyTextParser.cpp

```diff
@@ -216,6 +220,21 @@ bool LegacyTextParser::parse(const String& inText, std::u32string& outText,
     return true;
 }
 
+Sizef getRespectRatioSize(Sizef reqSize, const Sizef& orgSize) {
```

Comment:
1. Please follow the code style. A dash line before the function and a bracket on the new line when the function body begins.
2. Both Sizef args should be passed the same way. I prefer `const Sizef&` in this case. Using an argument as a temporary dest for internal calculations is not good, use local var instead.
3. Helper function in a module scope should be declared as `static`.

Reply:
fixed